test/metabase/notification/send_test.clj

```diff
@@ -118,3 +119,61 @@
                                                                                    [:timestamp :string]
                                                                                    [:message :string]]])}}
                       (t2/select-one :model/TaskHistory :task "channel-send"))))))))))
+
+(deftest send-notification-record-prometheus-metrics-test
+  (mt/with-prometheus-system! [_ system]
+    (notification.tu/with-notification-testing-setup
+      (mt/with-temp [:model/Channel chn notification.tu/default-can-connect-channel]
```

Comment:
nit: I prefer the abbreviation of just `ch` for channel (common in other cases)

Maybe you are worried about it looking like the core.async primitive though? Doesn't bother me personally.